Sources/Sentry/PrivateSentrySDKOnly~.mm

```diff
@@ -0,0 +1,384 @@
+#import "PrivateSentrySDKOnly.h"
```

Comment:
`l`: Why is this another file with `~` in the file name?

Reply:
Opsy, something wrong happening here.

Reply:
Weird stuff